Source/WebCore/platform/graphics/texmap/ClipPath.h

```diff
@@ -37,20 +37,21 @@ namespace WebCore {
 class ClipPath final {
     WTF_MAKE_TZONE_ALLOCATED(ClipPath);
 public:
+    ClipPath() = default;
     ClipPath(Vector<FloatPoint>&& vertices, unsigned bufferID, unsigned bufferOffsetInBytes);
 
     bool isEmpty() const { return m_vertices.isEmpty(); }
     unsigned bufferID() const { return m_bufferID; }
     const void* bufferDataOffsetAsPtr() const;
     unsigned numberOfVertices() const { return m_vertices.size(); }
 
-    FloatRect bounds() const { return m_bounds; }
+    const FloatRect& bounds() const { return m_bounds; }
 
 private:
 
     const Vector<FloatPoint> m_vertices;
-    unsigned m_bufferID;
-    unsigned m_bufferOffsetInBytes;
+    unsigned m_bufferID { 0 };
```

Comment:
In general it's good practice to separate the trivial style fixes in a PR that preceeds this one -- you can keep them here, just wanted to mention it.

Reply:
Ok, thanks for the hint